src/viam/sdk/components/camera.cpp

```diff
@@ -254,7 +255,7 @@ viam::component::camera::v1::DistortionParameters Camera::to_proto(
     return proto;
 }
 
-Camera::Camera(std::string name) : Component(std::move(name)){};
+Camera::Camera(std::string name) : Component(std::move(name)) {};
```

Comment:
This drives me crazy but sometimes vscode "format on save" will put a space here which then makes the linter complain, possibly due to mismatched versions of `clang-format`, you may just have to temporarily disable format on save or delete the space with another editor to get the build to pass